webpack/components/Content/Details/ContentDetails.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { Nav, NavItem, TabPane, TabContent, TabContainer, Grid, Row, Col } from 'patternfly-react';
+import { PropTypes } from 'prop-types';
+import { LoadingState } from '../../../move_to_pf/LoadingState/index';
+import ContentDetailInfo from './ContentDetailInfo';
+import ContentDetailRepositories from './ContentDetailRepositories';
+
+const ContentDetails = props => {
+  const { contentDetails, schema, displayMap} = props;
+  const {
+    loading, repositories,
+  } = contentDetails;
+
+  const tabHeaders = () => {
+    const children = props.children;
+    const tabs = schema.map((child, index) =>
+      <NavItem eventKey={index+3}>
+        <div>{child.tabHeader}</div>
+      </NavItem>
+    );
+    return (
+      tabs
+    );
+  };
+
+  const tabPanes = () => {
+    const children = props.children;
```

Comment:
what is props.children used for?

Reply:
I have to clean up some code from these commits.. :smile: